test/e2e/body.test.js

```diff
@@ -0,0 +1,16 @@
+const { assert, driver, By } = require('vl-ui-core').Test.Setup;
+const VlBodyPage = require('./pages/vl-body.page');
+
+describe('vl-body', async () => {
+    const vlBodyPage = new VlBodyPage(driver);
+
+    before(() => {
+        return vlBodyPage.load();
+    });
+
+    it('als gebruiker kan ik de inhoud van een body element zien', async () => {
```

Comment:
kunt ge niet ergens een element zetten dat stijl zou moeten krijgen en dan via "computed" style zien dat het die stijl  ook effectief heeft?

Reply:
Dat zal ik nog toevoegen, is inderdaad goede unit test.